features/borrower_confirms_deed.feature

```diff
@@ -0,0 +1,31 @@
+@borrower_confirms_deed
+
+Feature: Borrower confirms the Deed
+  As a Borrower
+  I want to confirm my mortgage deed
+  So that I can complete the remortgage process
+
+Background:
+  Given I have valid deed data with <1> borrowers
+  And I create the deed via the Deed API
+
+@us133a
+Scenario: Borrower enters a valid deed reference
```

Comment:
Feel the title of this scenario should reflect more the purpose of the test, this one looks a lot like the scenario name for borrower views the deed.